### PR TITLE
Minor changes to Portuguese translation and additional thoughts.

### PR DESCRIPTION
**Pull request by Hiers**
I'm sorry for the inconvinience.

### CODE REVIEW

**Comment by Hiers** [comment]
I'm starting to doubt the other translator, as this time some of the changes he made were factually incorrect and a duplicate translation was also made. I'm not sure why someone who seems to be a Brazilian Portuguese speaker would work on a non-Brazilian Portuguese translation, nor why he's doing Portuguese translations when he has some fairly glaring gaps in his understanding of it, nor why he would create a carbon copy of an existing translation, but I don't like having to police it.

File: static/locales/pt-PT.yaml [hunk 1]
```diff
@@ -182,15 +182,15 @@ Settings:
     Save Watched Progress: Guardar onde ficou da última vez num vídeo
     Clear Search Cache: Limpar Cache de Pesquisas
     Are you sure you want to clear out your search cache?: Tem a certeza de que quer
-      apagar o seu cache de pesquisas?
```
Comment (Hiers) [inline static/locales/pt-PT.yaml:185]: This change in particular is what makes me question him, as it was originally correct, but was changed to be incorrect.

**Comment by GilgusMaximus** [comment]
Okay I cannot say anything about the correctness of the translation as I do not speak Portuguese.
However as I've read in the matrix channel, there are many people on Weblate who just go to a project and start translating without any other connection towards the project. So it could be that someone just got the string offered and translated it.

**Comment by PrestonN** [comment]
As @GilgusMaximus alluded to.  There likely won't be much that can be done on our end to help.  I also do not speak the language so I have to trust our translators that the translations are accurate.

If the other translator that's making the changes continues to do so, I can only suggest that you attempt to reach out to them in order to get the issue resolved.  Weblate shows a history of all translated strings so it should be possible to find their contact information and reach out to them about it.

Either way, I appreciate you taking a serious look at the translations and making sure that they can be as accurate as possible.

**Comment by PrestonN** [comment]
Please solve your merge conflicts and then I will merge your changes.

**Comment by PrestonN** [comment]
Thank you!  I'll delete the other file myself if it isn't needed.

**Pull request merged**